diff --git a/setools/__init__.py b/setools/__init__.py
index 7b70f5e..4ccfa88 100644
--- a/setools/__init__.py
+++ b/setools/__init__.py
@@ -73,11 +73,11 @@ from .pcideviceconquery import PcideviceconQuery
 from .devicetreeconquery import DevicetreeconQuery
 
 # Information Flow Analysis
-from .infoflow import InfoFlowAnalysis
-from .permmap import PermissionMap
+#from .infoflow import InfoFlowAnalysis
+#from .permmap import PermissionMap
 
 # Domain Transition Analysis
-from .dta import DomainTransitionAnalysis
+#from .dta import DomainTransitionAnalysis
 
 # Policy difference
 from .diff import PolicyDifference
diff --git a/tests/__init__.py b/tests/__init__.py
index 75758e3..fc63689 100644
--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -22,12 +22,12 @@
 from . import commonquery
 from . import defaultquery
 from . import diff
-from . import dta
+#from . import dta
 from . import fsusequery
 from . import genfsconquery
 from . import ibendportconquery
 from . import ibpkeyconquery
-from . import infoflow
+#from . import infoflow
 from . import initsidquery
 from . import mlsrulequery
 from . import netifconquery
